src/index.js

```diff
@@ -56,27 +57,26 @@ function exists(file, done) {
  *
  * @param {String} base Current directory.
  * @param {String} file File path.
- * @param {String} ext File extension.
+ * @param {Array<String>} extensions Allowed file extensions.
  * @param {Function} next Completion callback.
  * @returns {void}
  * @private
  */
-function node(base, file, ext, next)  {
+function node(base, file, extensions, next)  {
   debug('Resolving file from node_modules: %s', file);
-  const extensions = [ext];
 
   return void resolve(file, {
     preserveSymlinks: false,
-    extensions: extensions,
-    basedir: base
+    basedir: base,
```

Comment:
You could just rename `base` to `basedir` and pass it in directly for consistency purposes.